packages/cron/src/jobs/metrics.js

```diff
@@ -14,6 +14,10 @@ const COUNT_USERS = 'SELECT COUNT(*) AS total FROM public.user'
 
 const COUNT_UPLOADS = 'SELECT COUNT(*) AS total FROM upload WHERE type = $1'
 
+const COUNT_TOTAL_UPLOADS = 'SELECT COUNT(*) AS total FROM upload'
+// This is a rolling 7 day date window of total uploads
+const REFRESH_UPLOAD_STATS = 'REFRESH MATERIALIZED VIEW upload_stats;'
```

Comment:
I think this needs `CONCURRENTLY` so that requests are not blocked while it is being updated.